src/encoded/static/components/objectutils.js

```diff
@@ -918,9 +945,10 @@ export function filterForVisualizableFiles(fileList) {
         return [];
     }
 
-    return fileList.filter((file) => isFileVisualizable(file));
-}
+    const files = fileList.filter((file) => isFileVisualizable(file));
 
+    return filterForVisualizableAssembly(files);
```

Comment:
This is a system-wide change that also affects carts. I remove all files from the displaying in Valis if they don't have a specific genome_assembly.